sys/net/gnrc/netif/lorawan/gnrc_netif_lorawan.c

```diff
@@ -28,20 +28,25 @@
 #include "net/gnrc/lorawan/region.h"
 #include "net/gnrc/netreg.h"
 
-#define ENABLE_DEBUG 0
+#define ENABLE_DEBUG 1
 #include "debug.h"
 
 #define MSG_TYPE_MLME_BACKOFF_EXPIRE (0x3458)           /**< Backoff timer expiration message type */
 
-static uint8_t _nwkskey[LORAMAC_NWKSKEY_LEN];
 static uint8_t _appskey[LORAMAC_APPSKEY_LEN];
+#if IS_USED(CONFIG_LORAWAN_1_1)
 static uint8_t _appkey[LORAMAC_APPKEY_LEN];
```

Comment:
why is `_appkey` not required in LoRaWAN 1.0.3?

Reply:
It is but its name has changed. The 1.1 spec uses the NwkKey in place for the AppKey.

Reply:
ok, I see. But maybe you could try to use the same variable names internally. Otherwise you cannot use regular `if (IS_ACTIVE())` stuff